src/routes/(admin)/questionbank/+page.server.ts

```
@@ -7,6 +7,7 @@ export const load: PageServerLoad = async ({ cookies, url }) => {
 	const token = getSessionTokenCookie();
 
 	let questionName = url.searchParams.get('name') || '';
+	let currentPage = url.searchParams.get('page') || 1;
```

ESLint

[error] 'currentPage' is never reassigned. Use 'const' instead.

(prefer-const)